libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -53,23 +53,21 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ACCTHRESH",    3,     AP_Arming,  accel_error_threshold,  AP_ARMING_ACCEL_ERROR_THRESHOLD),
 
-#if !APM_BUILD_TYPE(APM_BUILD_ArduCopter)
-    // @Param: MIN_VOLT
-    // @DisplayName: Minimum arming voltage on the first battery
+    // @Param: VOLT_MIN
+    // @DisplayName: Arming voltage minimum on the first battery
     // @Description: The minimum voltage on the first battery to arm, 0 disables the check.  This parameter is relevant for ArduPlane only.
```

Comment:
I agree pulling the battery checks out of board-voltage-checks makes a lot of sense.  That's an existing issue though so I've left it as is 'cuz I'm a little pressed for time and don't want to re-test.